FIPS/pom.xml

```diff
@@ -429,6 +429,10 @@
                        <pattern>org.jsoup</pattern>
                        <shadedPattern>${shadeBase}.org.jsoup</shadedPattern>
                      </relocation>
+                     <relocation>
```

Comment:
FIPS is a certification standard. We need to check whether zstd is allowed and won't break our compliance. 
@sfc-gh-jkaluzny We already use this library in GS. But I want to get your opinion on whether adding it to JDBC (and specifically to this FIPS file) has any consequences on our certifications?

Reply:
@sfc-gh-mhofman, @sfc-gh-pbulawa I would very much like to get you review and feedback for Ismail's comment. Thanks

Reply:
I think that we should be fine as the library is not doing any cryptographic operations thus it should not break the compliance.